docker/cgc_metadata_to_qiime_mapping_file.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, Gregory Poore.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# ----------------------------------------------------------------------------
+import numpy as np
+import pandas as pd
+import sevenbridges as sbg
+import glob, sys, getopt, os, json, unicodedata, argparse, time
+from pandas.io.json import json_normalize
+from tqdm import tqdm # progressbar
+from tqdm import trange
+
+parser = argparse.ArgumentParser(description="""
```

Comment:
Is there a reason to use `argparse` over `click`? I ask because `biocore` in general has standardized on `click` for CLI interface